com.unity.cinemachine/Tests/Runtime/CmColliderTests.cs

```diff
@@ -88,37 +80,30 @@ public IEnumerator CheckDampingWhenOccluded()
             m_Collider.DampingWhenOccluded = 1;
             var originalCamPosition = m_Vcam.State.GetFinalPosition();
             
-            yield return null; 
-            m_Brain.ManualUpdate();
+            yield return UpdateCinemachine();
             
             Assert.That(originalCamPosition, Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
 
             var obstacle = GameObject.CreatePrimitive(PrimitiveType.Cube);
             obstacle.transform.position = originalCamPosition; // place obstacle so that camera needs to move
 
-            yield return WaitForOnePhysicsFrame();
-            m_Brain.ManualUpdate();
+            yield return WaitForOnePhysicsFrame(); // ensure that moving the collider (obstacle) takes affect
+            yield return UpdateCinemachine();
```

Comment:
I wonder whether these 2 lines can be formalized in a baseclass method instead of always repeating this pattern with the same comment